src/SingleProject/Resizetizer/src/GenerateTizenManifest.cs

```diff
@@ -0,0 +1,253 @@
+﻿#nullable enable
+using System;
+using System.IO;
+using System.Linq;
+using System.Xml.Linq;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+namespace Microsoft.Maui.Resizetizer
+{
+	public class GenerateTizenManifest : Task
+	{
+		[Required]
+		public string IntermediateOutputPath { get; set; } = null!;
+
+		[Required]
+		public string TizenManifestFile { get; set; } = TizenManifestFileName;
+
+		public string GeneratedFilename { get; set; } = TizenManifestFileName;
+
+		public string? ApplicationId { get; set; }
+
+		public string? ApplicationDisplayVersion { get; set; }
+
+		public string? ApplicationVersion { get; set; }
+
+		public string? ApplicationTitle { get; set; }
+
+		public ITaskItem[]? AppIcon { get; set; }
+
+		public ITaskItem[]? SplashScreen { get; set; }
+
+		[Output]
+		public ITaskItem GeneratedTizenManifest { get; set; } = null!;
+
+		const string TizenManifestFileName = "tizen-manifest.xml";
+
+		const string IconDefaultDpiType = "xhdpi";
+
+		const string SharedResourcePath = "shared/res/";
+
+		const string UiApplicationName = "ui-application";
+
+		const string PackageName = "package";
+
+		const string AppidName = "appid";
+
+		const string VersionName = "version";
+
+		const string LabelName = "label";
+
+		const string IconName= "icon";
+
+		const string SplashScreensName = "splash-screens";
+
+		const string SplashScreenName = "splash-screen";
+
+		const string DpiName = "dpi";
+
+		bool _shouldUpdateOriginalManifest;
+
+		string? _tizenManifestFilePath;
+
+		public override bool Execute()
+		{
+			try
+			{
+				Directory.CreateDirectory(IntermediateOutputPath);
+
+				_tizenManifestFilePath = Path.Combine(Environment.CurrentDirectory, TizenManifestFile);
+
+				var targetFilename = Path.Combine(IntermediateOutputPath, GeneratedFilename);
+
+				var manifest = XDocument.Load(_tizenManifestFilePath);
+
+				UpdateManifest(manifest);
+
+				manifest.Save(targetFilename);
+
+				GeneratedTizenManifest = new TaskItem(targetFilename);
+			}
+			catch (Exception ex)
+			{
+				Log.LogErrorFromException(ex);
+			}
+
+			return !Log.HasLoggedErrors;
+		}
+
+		void UpdateManifest(XDocument tizenManifest)
+		{
+			var xmlns = tizenManifest.Root!.GetDefaultNamespace();
+			var manifest = tizenManifest.Root;
+			var uiApplication = manifest.Element(xmlns + UiApplicationName);
+
+			if (manifest == null || uiApplication == null)
+			{
+				return;
+			}
+
+			UpdateSharedManifest(xmlns, manifest);
+
+			UpdateOriginalManifest(tizenManifest);
```

Comment:
Is there a reason to update the manifest in the user directory?

Reply:
It probably is not the best idea to update back to the original manifest in incremental build perspective. However I'd like to minimize the confusion about the difference between the manifest and actual packaged manifest to Tizen developers by updating application id, version and title back to the manifest when they are changed.

Reply:
Does this new task support the user overriding the values in the manifest? I think that is more the issue here. If the tool overwrites the file, then if it makes a mistake or the user wants something different they can't do that.

For example, if some designer comes and gives the dev a set of files to copy directly into the platform folder and the dev adds some more icons - effectively adding 2 icons into the app: 1 from the designer and 1 from MauiIcon. Is there a way for the dev to prefer the specific icon and overwrite all the values?

For the other platforms, if there is no icon set in the manifest, then the tool provides one. If there is an icon, then the tool does nothing. This allows for the tool to only do work if the dev has not explicitly told it not to. The dev should always have the last say - even if they are wrong. :)

It may just be the case that the new template manifest has nothing in it so the tool always generates the file.

Reply:
Now Tizen resizetizer does not support user overriding values in the manifest.  It only allows the values that tool does not provide, so basically it is the tool first policy for now or I would say maui resources first.
It totally makes sense what other platforms do like you said and I have been actually thinking how Tizen can change the behavior like that. 
The thing is Tizen manifest editor now shows warnings if those values are empty or other special values. I think I would need to ask to fix that first or find other way.

Reply:
Our manifest editor would also think `$placeholder$` is not a proper value. That was the thing. I will try applying it though. Thanks

Reply:
Yes, I'm doing it with something like `maui-application-id-placeholder`. Thanks :)

Reply:
Done updating to use placeholder values on tizen-manifest.xml to give priority to developer's input 😄 and no update back to the tizen-manifest.xml in working folder.

Reply:
The developers are always right, even when they are wrong.